pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -870,14 +869,15 @@ func (b *changefeedResumer) OnPauseRequest(
 // getQualifiedTableName returns the database-qualified name of the table
 // or view represented by the provided descriptor.
 func getQualifiedTableName(
-	ctx context.Context, execCfg sql.ExecutorConfig, txn *kv.Txn, desc catalog.TableDescriptor,
+	ctx context.Context, execCfg *sql.ExecutorConfig, txn *kv.Txn, desc catalog.TableDescriptor,
```

Comment:
It's weird that the execCfg was being dereferenced, and it doesn't seem like there was a reason for it.

Reply:
Agreed